src/PrestaShopBundle/Resources/config/services/bundle/twig.yml

```diff
@@ -118,3 +118,13 @@ services:
       class: 'PrestaShopBundle\Twig\Extension\NumberExtension'
       tags:
         - { name: twig.extension }
+
+    prestashop.bundle.twig.filesystem_loader:
+        class: 'Twig\Loader\FilesystemLoader'
+        arguments:
+            - '%kernel.project_dir%/'
```

Comment:
```suggestion
            - '%kernel.project_dir% ~ !php/const DIRECTORY_SEPARATOR'
```

Reply:
This brings absolutely nothing but complexity 🤔  (and no Windows is not a valid argument any more since it handles / very well)

Reply:
Woot ? Really for Windows ?

Reply:
yep! @PierreRambaud can confirm it, I believe he's the one who explained it to me

Reply:
https://alanhogan.com/tips/php/directory-separator-not-necessary It has always been useless to use DIRECTORY_SEPARATOR